argus/resources/base_cloudbaseinit.conf

```diff
@@ -0,0 +1,28 @@
+[DEFAULT]
```

Comment:
I think that the config file should be named `cloudbaseinit.conf-sample` (or template).